src/Serilog.Sinks.Http/Serilog.Sinks.Http.csproj

```diff
@@ -58,13 +58,21 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net45'">
+    <DefineConstants>$(DefineConstants);NET45</DefineConstants>
+  </PropertyGroup>
+
   <PropertyGroup Condition="'$(TargetFramework)' == 'net45' Or '$(TargetFramework)' == 'net461'">
     <DefineConstants>$(DefineConstants);HRESULTS</DefineConstants>
   </PropertyGroup>
 
   <!-- .NET STANDARD -->
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
+    <DefineConstants>$(DefineConstants);NETSTANDARD_1_3</DefineConstants>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461' Or '$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
What do you think about inverting this statement, saying that we add `Microsoft.Extensions.Configuration` if we're not in target framework `net45` or `netstandard1.3`. That way we don't have to remember to update this statement when we add support for future target frameworks.

What do you think, would it be beneficial?

Reply:
Yes, that makes more sense. I'll update the branch.

Reply:
I've just inverted the condition and amended the commit